app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all
+    @chefs_ingredients = Ingredient.chefs_ingredients
```

Comment:
I would have expected this to be more something like `@chef.all_ingredients`, because otherwise, how do you know which chef you're pulling ingredients for? Because your route to hit this action is nested, you have access to the chef id